editor/components/post-type-support-check/index.js

```diff
@@ -16,7 +16,9 @@ import { compose } from '@wordpress/element';
 import { getCurrentPostType } from '../../store/selectors';
 
 function PostTypeSupportCheck( { postType, children, supportKey } ) {
```

Comment:
Maybe the supportKey should be named supportKeys given that we accept more than one.